src/lib/annotations/doc/CreateHighlightDialog.js

```diff
@@ -309,6 +314,13 @@ class CreateHighlightDialog extends EventEmitter {
         highlightDialogEl.classList.add(CLASS_CREATE_DIALOG);
         highlightDialogEl.innerHTML = CREATE_HIGHLIGHT_DIALOG_TEMPLATE;
 
+        // Get rid of the caret
+        if (this.isMobile) {
+            highlightDialogEl.classList.add('bp-mobile-annotation-dialog');
+            highlightDialogEl.classList.add('bp-annotation-dialog');
+            highlightDialogEl.querySelector('.bp-annotation-caret').remove();
```

Comment:
Can we have this togglable in the HTML itself? something like `${this.isMobile ? CLASS_HIDDEN : '' }`